File: By-Swift/By-Swift/Leetcode/squared-of-a-sorted-array.swift
// Runtime: 216 ms, faster than 95.30% of Swift online submissions for Squares of a Sorted Array.
// Memory Usage: 15.5 MB, less than 66.44% of Swift online submissions for Squares of a Sorted Array.
class Solution {
    func sortedSquares(_ nums: [Int]) -> [Int] {
        return nums.map{Int(pow(Double($0),2))}.sorted()
    }
}

File: By-Python/leetcode/squared-of-a-sorted-array.py
# Runtime: 232 ms, faster than 67.44% of Python3 online submissions for Squares of a Sorted Array.
# Memory Usage: 15.8 MB, less than 91.16% of Python3 online submissions for Squares of a Sorted Array.

class Solution:
    def sortedSquares(self, nums: List[int]) -> List[int]:
        return sorted(list(map(lambda x: x**2, nums)))
        